### ISSUE TITLE
Sorting by local secondary index not possible

### ISSUE THREAD

**Issue by Igor-G** [derjust/spring-data-dynamodb#86]
Sorting by the local secondary index doesn't seem to be possible at the moment. I tracked the issue and found some limitation that only sorting by the primary range key is allowed.

This [commit](https://github.com/Igor-G/spring-data-dynamodb/commit/b970a7ab0ea171a5a585d961286dc63e8af05538) solves the issue. But it is not a clean solution as it has one failing test ignored. If someone is willing to discuss the solution, additional changes required, and why the test fails (from the dynamoDB spec, this is a valid query), I can clean-up the fix and issue a PR.

**Comment by derjust** [comment]
Yes please feel to go ahead providing more input or even an PR 👍
